src/supertux/sector.cpp

```diff
@@ -394,9 +413,62 @@ Sector::draw(DrawingContext& context)
 {
   BIND_SECTOR(*this);
 
-  Camera& camera = get_camera();
+#if 0
+  context.push_transform();
```

Comment:
What is this region for?

Reply:
An attempt at making a split-screen camera. I left it there for future maintainers, so they have a working example, in case I happen to get bus-factored. I can remove it if preferred.